qiskit_experiments/library/randomized_benchmarking/interleaved_rb_experiment.py

```diff
@@ -140,12 +141,12 @@ def __init__(
             full_sampling=full_sampling,
         )
         # Convert interleaved element to integer for speed
-        if self.num_qubits <= 2:
+        if self.num_qubits in {1, 2}:
```

Comment:
It looks like we don't need to nest if statements here:
```
interleaved_circ = self._interleaved_elem.to_circuit()
if self.num_qubits == 1:
    ...
elif self.num_qubits == 2:
    ...
```

I'm wondering what the plans/status of self.num_qubits > 2? Does the non-existence of `self._interleaved_elem` in this case cause an error?

Reply:
For `self.num_qubits > 2`, the corresponding Clifford object is stored in `self._interleaved_elem`. I've updated the code to make that clearer in aff3fb5.